qemu/tests/hv_kvm_unit_test.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import re
+import json
+
+from virttest import error_context
+from virttest import data_dir
+from virttest import cpu
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run kvm-unit-tests for Hyper-V testdev device
+
+    1) compile kvm-unit-tests tools source code
+    2) Run each unit tests by compiled binary tools
+    3) For each unit test, compare the test result to expected value
+    """
+    tmp_dir = data_dir.get_tmp_dir()
+    kvm_unit_tests_dir = data_dir.get_deps_dir("kvm_unit_tests")
+    compile_cmd = params["compile_cmd"] % (tmp_dir, kvm_unit_tests_dir)
+    test_cmd = params["test_cmd"]
+    unit_tests_mapping = params["unit_tests_mapping"]
+    skip_tests = params.get("skip_tests", "").split()
+    cpu_flags = params["cpu_model_flags"]
+    cpu_model = cpu.get_qemu_best_cpu_model(params)
+    cpu_param = cpu_model + cpu_flags
+
+    error_context.context("Copy & compile kvm-unit-test tools", logging.info)
+    process.system(compile_cmd, shell=True)
+
+    error_context.context("Run unit tests", logging.info)
+    for unit_test, unit_test_result in json.loads(unit_tests_mapping).items():
+        if unit_test in skip_tests:
+            pass
```

Comment:
`pass` is a null operation — when it is executed, nothing happens. so you wanna skip the following routine to use `continue`.